api/services/GithubBuildStatusReporter.js

```diff
@@ -98,7 +98,7 @@ const reportBuildStatus = (build) => {
       options = {
         owner: site.owner,
         repo: site.repository,
-        sha: build.commitSha,
+        sha: build.cloneCommitSha || build.webhookCommitSha,
```

Comment:
Hmmm, how does this work right now? Does Github show the latest reported status OR the reported status for the latest commit included in the PR? (looking at the API docs, I wonder if they are using https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#get-the-combined-status-for-a-specific-reference which would seem to be just the latest) Before this change I could see this being a bit wonky in that we may report a status for a different (out of date) commit!

This seeeeeems fine?

Makes me think we *really* should build specific commits...